### PR TITLE
Log command used to start metals

### PR DESCRIPTION
**Pull request by kpbochenek**
Output:
```
Java home: /usr/lib/jvm/default
Starting metals using command '/usr/lib/jvm/default/bin/java' with args '-Xss4m,-Xms100m,-Dmetals.client=vscode,-classpath,/home/kp...
```

### CODE REVIEW

**Comment by kpbochenek** [comment]
@tgodzik logging a java path and arguments used could be helpful debugging issues like https://github.com/scalameta/metals-vscode/issues/366 ? WDYT?

Extension internally constructs a command to be executed from cmd and args so I can't log exact command but maybe this will be helpful enough?

Not 100% sure about this that's why 'draft' and waiting for your opinion

File: src/extension.ts [hunk 1]
```diff
@@ -311,6 +311,14 @@ function launchMetals(
     clientOptions
   );
 
+  {
```
Comment (tgodzik) [inline src/extension.ts:314]: Does it need to be inside brackets?

Comment (kpbochenek) [reply src/extension.ts:314]: nope, just scoping cmd and args

**Comment by tgodzik** [comment]
I think we don't necessarily need this now. Users can easily check the java process if needed. I will close it for now and reopen if more people want this change.

**Pull request closed without merging**